doc/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+  version: 1.0.0
+
+servers:
+  - url: https://pkimet.al/
+    description: Stable API server
+  - url: https://dev.pkimet.al/
+    description: Development API server
+  - url: http://localhost:8080/
+    description: Local API server
+
+paths:
+  /lintcert:
+    post:
+      summary: Lints a X.509 certificate
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+  /linttbscert:
+    post:
+      summary: Creates and lints a X.509 certificate from the specified TBSCertificate
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+  
+  /lintcrl:
+    post:
+      summary: Lints a X.509 certificate revocation list (CRL)
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+  /linttbscrl:
+    post:
+      summary: Creates and lints a X.509 certificate revocation list (CRL) from the specified TBSCertList
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+
+  /lintocsp:
+    post:
+      summary: Lints an OCSP response
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+  /linttbsocsp:
+    post:
+      summary: Creates and lints an OCSP response from the specified ResponseData
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+          
+  /profiles:
+    get:
+      summary: Retrieves the profiles available for linting
+      responses:
+        '200':
+          description: A list of profiles available for linting
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/LintProfile'
+                  
+  /linters:
+    get:
+      summary: Retrieves the linters available for linting
+      responses:
+        '200':
+          description: A list of linters available for linting
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Linter'
+
+components:
+  requestBodies:
+    LintRequestBody:
+      description: The parameters for the linting request
+      required: true
+      content:
+        application/x-www-form-urlencoded:
+          schema:
+            $ref: '#/components/schemas/LintRequest'
+
+  responses:
+    200LintingSuccessful:
```

Comment:
This will affect the models that are generated by some OpenAPI generators. I'd strongly suggest sticking to `LintingSuccessful` or something else that's a bit more declarative.

I'd also consider running this through https://stoplight.io/open-source/spectral

Reply:
> I'd also consider running this through https://stoplight.io/open-source/spectral

Thanks for the suggestion, @sigmavirus24.  I've run @CBonnell's latest `openapi.yaml` through Stoplight, which flagged a number of warnings.  But I'll merge this PR as-is, then follow up on those warnings in a new PR.

My Stoplight workspace is publicly accessible at https://pkimetal.stoplight.io/docs/pkimetal/68a341c676710-pkimetal-api.